blaseball_mike/models/team.py

```diff
@@ -59,42 +35,38 @@ def load_at_time(cls, id_, time):
         """
         Load blaseball team with roster at given datetime.
         """
-        if isinstance(time, str):
-            time = parse(time)
-
-        team = list(chronicler.get_entities("team", id_, at=time))
-        if len(team) == 0:
-            return None
-        return cls(dict(team[0]["data"], timestamp=time))
+        warnings.warn("instead of .load_at_time(id_, time), use .load_one(id_, time=time)",
+                      DeprecationWarning, stacklevel=2)
+        return cls.load_one(id_, time=time)
 
     @Base.lazy_load("_lineup_ids", cache_name="_lineup", default_value=list())
     def lineup(self):
         if getattr(self, "timestamp", None):
-            return [Player.load_one_at_time(x, self.timestamp) for x in self._lineup_ids]
+            return [Player.load_one(x, time=self.timestamp) for x in self._lineup_ids]
         else:
             players = Player.load(*self._lineup_ids)
             return [players.get(id_) for id_ in self._lineup_ids]
```

Comment:
Now that we can use the same load function for both, we dont need these if/else trees